### PR TITLE
DynamoDB Streams

### PR DESCRIPTION
**Pull request by kiril-pirozenko-home24**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (mariusgrigaitis) [inline .vscode/launch.json]: ??? what's this mess?

File: actions/createTable.js [hunk 2]
```diff
@@ -67,5 +71,21 @@ module.exports = function createTable(store, data, cb) {
       })
     })
   })
+}
+
+function toStreamLabelString(date) {
```
Comment (mariusgrigaitis) [inline actions/createTable.js]: Why not take zulu time and remove zulu part?

File: actions/putItem.js [hunk 3]
```diff
@@ -29,7 +31,49 @@ module.exports = function putItem(store, data, cb) {
 
           itemDb.put(key, data.Item, function(err) {
             if (err) return cb(err)
-            cb(null, returnObj)
+
+            if (table.LatestStreamArn) {
```
Comment (mariusgrigaitis) [inline actions/putItem.js:34]: This code is (almost) duplicated with delete / put actions

**Review by mariusgrigaitis — CHANGES_REQUESTED** [review]

**Comment by kiril-pirozenko-home24** [comment]
Closing in favour of https://github.com/mhart/dynalite/pull/67

**Pull request closed without merging**